src/components/CreepypastaForm.js

```diff
@@ -4,20 +4,28 @@ import { useState } from 'react';
 import { nanoid } from 'nanoid';
 import styled from 'styled-components';
 
-export default function CreepypastaFrom({ handleCreateCreepypasta }) {
-  const [wordCount, setWordCount] = useState(0);
-  const [counter, setCounter] = useState(20);
+export default function CreepypastaFrom({
```

Comment:
i think the name of the function is meant CreepypastaForm instead of CreepypastaFrom, but maybe not in the ghost world :-))

Reply:
You are right! This is my go to typo